LibraBFT/Abstract/BFT.agda

```diff
@@ -85,7 +97,7 @@ module LibraBFT.Abstract.BFT
    ...| tri> ¬a ¬b c = slist
 
 
-   trans-OnHead : ∀ {xs : List Member} {y x : Member}
+   trans-OnHead : ∀ {n} {xs : List (Fin n)} {y x : Fin n}
```

Comment:
There are other lemmas that can go to `LibraBFT.Lemmas`, and maybe as we have said we should create a directory with separate files, one for lemmas about lists, other with arithmetic lemmas, etc. We already have a `TODO` for that. Maybe it's better to do those changes when addressing that `TODO`. 

I don't agree with you. I think it makes sense to be in a separate file. Even the definition of union and intersect maybe should go to a different file. I would leave in the BFT module only the necessary lemmas and definition to prove the `bft-lemma`. Otherwise, I think it adds too much entropy for someone trying to analyse the proof.